schema/rpc.fbs

```diff
@@ -338,20 +344,71 @@ table AutoBoneProcessRequest {
 
 table AutoBoneProcessStatusResponse {
     process_type: AutoBoneProcessType;
-    message: string;
```

Comment:
this breaks compatibility. Instead we could mark with deprecated attribute instead, which prevents code generation but doesn't break binary format.

Is there anything that would break if we changed this?

Reply:
I thought it was totally unused, but apparently GUI logged it though it already checks if it's null, so it's not useful whatsoever really. Uriel mentioned it might cause deserialization issues though?

Reply:
checking its null is not the issue - by removing the field, all subsequent fields in the table will be wrong - think of the position in the table representing the index into an array, and we removed an element in the array, so subsequent fields will have an off-by-one error. So its a breakage of not just this field, but all subsequent fields.